view/partial/remember-patient-pin.hbs

```diff
@@ -0,0 +1,23 @@
+<div class="modal" id="remember-patient-dialog" role="dialog">
+    <div class="modal-dialog">
+        <div class="modal-content">
+            <div class="modal-header">
+              <button type="button" class="close" data-dismiss="modal">
+                  <span>
+                      &times;
+                  </span>
+                  <span class="sr-only">
+                      Close
+                  </span>
+              </button>
+              <h4 class="modal-title">Patient {{ patient.pin }} Created!</h4>
+            </div>
+            <div class="modal-body">
+              <p>Please copy the pin number {{ patient.pin }} down so information can be re-identified later.</p>
+            </div>
+            <div class="modal-footer">
+              <button type="button" class="btn btn-default" data-dismiss="modal">Close</button>
```

Comment:
move text on line 13, 16 and 19, onto a separate newline from tags. (e.g. how lines 9-11 are)